src/reconcile/rec_write.c

```diff
@@ -2323,6 +2324,13 @@ __rec_build_delta(
         if (WT_BUILD_DELTA_LEAF(session, r)) {
             WT_RET(__rec_build_delta_leaf(session, full_image, r));
             *build_deltap = true;
+            WT_STAT_CONN_DSRC_INCR(session, rec_page_delta_leaf);
+        }
+    } else if (F_ISSET(r->ref, WT_REF_FLAG_INTERNAL)) {
+        /* The internal page delta would have already been built at this point if one exists. */
+        if (r->delta.size > 0) {
+            *build_deltap = true;
+            WT_STAT_CONN_DSRC_INCR(session, rec_page_delta_internal);
```

Comment:
This should be increase in here
```
    /* Write the disk image and get an address. */
    if (build_delta) {
        /* We must only have one delta. Building deltas for split case is a future thing. */
        WT_ASSERT(session, last_block);
        WT_ASSERT(session, block_meta->checkpoint_id >= WT_DISAGG_CHECKPOINT_ID_FIRST);
        multi->block_meta = *block_meta;
        WT_ACQUIRE_READ(checkpoint_id, conn->disaggregated_storage.global_checkpoint_id);
```

Reply:
Otherwise, we may still set build_delta to false in here:

```
if (last_block && r->multi_next == 1 && block_meta->page_id != WT_BLOCK_INVALID_PAGE_ID &&
      block_meta->delta_count < btree->max_consecutive_delta) {
        WT_RET(__rec_build_delta(session, r, chunk->image.mem, &build_delta));
        /* Discard the delta if it is larger than one tenth of the size of the full image. */
        if (build_delta && ((r->delta.size * 100) / chunk->image.size) > btree->delta_pct)
            build_delta = false;
    }
```

Reply:
Updated here: [0c89a3a](https://github.com/wiredtiger/wiredtiger/pull/11671/commits/0c89a3acbfbb57d025bcf67144cddde9466010d8)